README.md

```diff
@@ -16,12 +16,43 @@ Install Bun JS runtime dependencies `curl -fsSL https://bun.sh/install | bash`
 
 Install JS dependencies `bun install`
 
-### Running
+## Running
 
-<i>Running the app from vscode's terminal can lead to an error, try running from your system terminal.</i>
+### Running with just
+
+You can use `just` to run this project automatically. Install `just` with:
+
+```sh
+cargo install just # using cargo
+
+# or, alternatively
+apt install just # for debian & ubuntu
+brew install just # for MacOS
+```
+
+You'll need to add a `.env` file that includes a path to your local copy of the internal service repo:
+
+```sh
```

Comment:
We should allow for trailing `/` or `\` on windows. In the code, we just do `<&str>::ends_with()`